src/managed_blmo.jl

```diff
@@ -34,12 +34,12 @@ mutable struct ManagedBoundedLMO{SBLMO<:SimpleBoundableLMO} <: BoundedLinearMini
     simple_lmo::SBLMO
     lower_bounds::Vector{Float64}
     upper_bounds::Vector{Float64}
-    n::Int
     int_vars::Vector{Int}
+    n::Int
     solving_time::Float64
 end
 
-function ManagedBoundedLMO(simple_lmo, lb, ub, n, int_vars)
+function ManagedBoundedLMO(simple_lmo, lb, ub, int_vars, n)
```

Comment:
since we are changing the order of arguments, this is technically a breaking change. Can we add a constraint in the signature here that `int_vars` is a vector of Int and `n` is an Int? To be sure the constructor doesn't get called with the old signature?

Reply:
Okay, yes. I believe this is already the case for the struct but I'll add it to the function as well.